### PR TITLE
OHIF-79: Study Browser side panel

### PR DESCRIPTION
**Pull request by galelis**
**OHIF-79**

### Description
UI implementation using mockdata for StudyBrowser, designed to be the content of a sidePanel. 

### Preview
Apr-10-2020 16-44-26

### CODE REVIEW

File: platform/ui/src/components/SidePanel/SidePanel.jsx [hunk 1]
```diff
@@ -65,10 +65,11 @@ const SidePanel = ({
             variant="text"
             color="inherit"
             rounded="none"
+            style={{ height: '52px' }}
```
Comment (rodrigolabs) [inline platform/ui/src/components/SidePanel/SidePanel.jsx:68]: This is very specific? There are many height classes that maybe could be used, as `height` extends from `spacing`:

https://github.com/OHIF/Viewers/blob/92805dd9b6f31f51cd58705597ea711b2f54e9d5/platform/ui/tailwind.config.js#L152-L174

File: platform/ui/src/components/StudyBrowser/StudyBrowser.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                  instances={instances}
+                  modalities={modalities}
+                  trackedSeries={trackedSeries}
+                  isActive={isActive}
+                  onClick={() => {
+                    if (isActive) {
+                      setStudyActive(null);
+                    } else {
+                      setStudyActive(studyInstanceUid);
+                    }
```
Comment (rodrigolabs) [inline platform/ui/src/components/StudyBrowser/StudyBrowser.jsx:76]: Just a cleaner suggestion:

```suggestion
setStudyActive(isActive ? null : studyInstanceUid);
```

File: platform/ui/src/components/Thumbnail/Thumbnail.jsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        <div
+          className={classnames(
+            'flex flex-col items-center justify-start p-2 mb-2 relative cursor-pointer',
+            isTracked && 'rounded-sm hover:bg-gray-900 showTooltipOnHover'
+          )}
+        >
+          <Icon
+            name={trackedIcon}
+            className="text-primary-light mb-2"
+            style={{ width: '19px' }}
```
Comment (rodrigolabs) [inline platform/ui/src/components/Thumbnail/Thumbnail.jsx:33]: Same as height (above), if possible and if this is not very specific, we can maybe replace this for `w-5`.

File: platform/ui/src/components/Thumbnail/Thumbnail.jsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+            )}
+          >
+            <div className="flex flex-row flex-1">
+              <div className="flex flex-col flex-1 pr-4">
+                <span>
+                  Series is <span className="text-white">tracked</span>
+                </span>
+                {viewportIdentificator && (
+                  <span>
+                    in viewport{'  '}
```
Comment (rodrigolabs) [inline platform/ui/src/components/Thumbnail/Thumbnail.jsx:50]: This `{'  '}` was a linter fix or an intended space? If it's a space, I'd prefer to use `mr-1` or `mr-2`

**Comment by rodrigolabs** [comment]
@galelis Great job, man! I'm loving how the UI is progressing well. I left some small comments.

**Comment by galelis** [comment]
All callouts fixed @rodrigolabs thanks sir! :)

**Comment by rodrigolabs** [comment]
> All callouts fixed @rodrigolabs thanks sir! :)

Awesome -- LGTM ✅

**Pull request merged**